yang/vyatta-protocols-frr-bgp-v1.yang

```diff
@@ -1521,6 +1583,95 @@ module vyatta-protocols-frr-bgp-v1 {
 							}
 						}
                                         }
+
+					container ipv4-vpn {
+						description "BGP neighbor parameters";
+						presence "Enables BGP neighbor IPv4 parameters";
+						configd:help "BGP neighbor parameters";
+						uses peer-grp-neighbor-afi-common-settings {
+							refine filter-list/export {
+								must "not(../../peer-group)" {
+									error-message "You may not configure filter-list export for a neighbor in peer-group\n";
```

Comment:
IETF max line length is 73 characters, and in DANOS we generally stick to under 80 characters for YANG files so you don't end up having to scroll sideways to read some lines (which really slows down reviews).  You also lose the context of the surrounding lines if they disappear off the left of the screen.

Furthermore, in this case you have a set of multiple changes close enough together that the side-scroll bar is miles down the page, making it incredibly difficult to scroll sideways.  I have to page down, scroll sideways, then page back up again.

Reply:
We understand the issue. 
We tried to be consistent with the existing file, so we did it in that manner. 

Now we have tried to rectify line length our part of code. Please have a look and suggest.